fastlane/spec/env_spec.rb

```diff
@@ -4,9 +4,8 @@
 describe Fastlane do
   describe Fastlane::EnvironmentPrinter do
     before do
-      stub_request(:post, %r{https:\/\/refresher.fastlane.tools\/.*}).
```

Comment:
Great question. I was thinking about this, and the reason why this wasn't detected by the tests is that we have a `rescue` block for internet connection failures that would just append the text `Check failed`